tensor2tensor/bin/t2t_translate_all.py

```diff
@@ -67,6 +67,7 @@
 
 
 def main(_):
+  FLAGS = flags.FLAGS
```

Comment:
FLAGS defined above. rm

Reply:
I can delete the current line 41 (I hope), but not this one.
FLAGS need to be a local variable because they are used few lines below -- https://github.com/martinpopel/tensor2tensor/blob/61005b0b1e063e3e26cb403cf6e8cdea3509c3b5/tensor2tensor/bin/t2t_translate_all.py#L94
Thanks to `**locals()`, the users can provide e.g `--decoder_command='qsub my-decoder.sh {params} --my_param={FLAGS.my_param}'` and I think it is more user-friendly than forcing them to write `--my_param={flags.FLAGS.my_param}`.

BTW: Last time someone deleted this line during an internal-Google refactor, but didn't noticed me within PR review as you do now. It was one of the reasons why `t2t_translate_all.py` didn't work (another reason was I was lazy to write a test).

Reply:
I see. How about this?:

```
locals_and_flags = dict()
local_and_flags.update(FLAGS.__dict__)
locals_and_flags.update(locals())
...
.format(**locals_and_flags)
```

Reply:
OK. If you feel it's more elegant, I will do it this way (and test).